urdf/package.xml

```diff
@@ -38,6 +40,7 @@
   <exec_depend>urdfdom_headers</exec_depend>
 
   <build_export_depend>pluginlib</build_export_depend>
+  <build_export_depend>rcutils</build_export_depend>
```

Comment:
Same here; we don't need to `build_export_depend` here, since they are PRIVATE dependencies:

```suggestion
```

Reply:
https://github.com/ros2/urdf/pull/37/commits/e9419ebdf3a3fe600e587a0e437f66b56756ee74